### ISSUE TITLE
Parsing warning FYI for 2.7.0.rc1

### ISSUE THREAD

**Issue by rhuppert** [mikel/mail#1115]
On my development system, I'm using rails 5.03, ruby 2.4, sidekiq 5.0, and premailer-rails 1.96, with a vanilla implementation of ActiveJob and ActionMailer.

**I'm using mail 2.7.0.rc1**

Sidekiq generates the following warning 105 times on outgoing email:
```
2017-05-26T22:04:43.784Z 27193 TID-gqne3mkwo ActionMailer::DeliveryJob JID-29e4d82779187041b6c3d80c INFO: start
WARNING: Could not parse (and so ignoring) 'Content-Type: text/plain'
WARNING: Could not parse (and so ignoring) 'Content-Type: text/plain'
WARNING: Could not parse (and so ignoring) 'Content-Type: text/plain'
2017-05-26T22:04:46.331Z 27193 TID-gqne3mkwo ActionMailer::DeliveryJob JID-29e4d82779187041b6c3d80c INFO: done: 2.547 sec
```
The plain text version does get delivered.

**When I reverted to 2.6.5, the warnings went away.**

Just thought you might like to know.

**Comment by jeremy** [comment]
In 2.7, warnings use `Kernel#warn` instead of writing to `$stderr` directly. Sounds like this may be the difference you're seeing: the warnings were present in 2.6.5 but weren't visible in logs, etc. Have you dug any further?

**Comment by jeremy** [comment]
(See #1105)

**Comment by rhuppert** [comment]
I created an issue on Sidekiq and as per @mperham:
> If it generates the same warning with deliver_now instead of deliver_later then it has nothing to do with Sidekiq.

The only place I saw the warning was in the sidekiq console and, of course there was no warning when I bypassed background processing with deliver_now. I saw no warnings in my development log.

I would dig further, but it may be over my head already?  Can I assume this might be a Sidekiq issue? I can pass it on, if so.
